keras/backend/tensorflow/numpy.py

```diff
@@ -783,6 +788,126 @@ def prod(x, axis=None, keepdims=False, dtype=None):
     return tfnp.prod(x, axis=axis, keepdims=keepdims, dtype=dtype)
 
 
+def _quantile(x, q, axis=None, method="linear", keepdims=False):
+    # ref: tfp.stats.percentile
+    # float64 is needed here and below, else we get the wrong index if the array
+    # is huge along axis.
+    q = tf.cast(q, "float64")
+
+    # Move `axis` dims of `x` to the rightmost, call it `y`.
+    if axis is None:
+        y = tf.reshape(x, [-1])
+    else:
+        x_ndims = tf.TensorShape(x.shape).rank
+
+        # _make_static_axis_non_negative_list
+        _axis = np.array(axis, dtype="int32")
```

Comment:
IMO either we should `tfnp` or `tf` for this

Reply:
Hi @AakashKumarNain 
I have rewritten the logic to use pure python built-ins.

No `import numpy as np` now